File: app/src/main/java/com/example/climbingdiary/models/Styles.java
package com.example.climbingdiary.models;

public class Styles {
    public static String[] getStyle(Boolean UpperCase){
        if(UpperCase){
            return new String[]{"OS","RP","FLASH"};
        }else{
            return new String[]{"os","rp","flash"};
        }
    }
}

File: app/src/main/java/com/main/climbingdiary/model/Styles.kt
package com.main.climbingdiary.model

import java.util.*

object Styles {

    fun getStyle(UpperCase: Boolean): Array<String> {
        return if (UpperCase) {
            arrayOf("OS", "RP", "FLASH")
        } else {
            arrayOf("os", "rp", "flash")
        }
    }

    fun getStyleRatingFactor(style: String): Int {
        return listOf(*getStyle(false))
            .indexOf(style.toLowerCase(Locale.ROOT)) * 5
    }
}